### ISSUE TITLE
Ability to add multiple piholes

### ISSUE THREAD

**Issue by AquaRelliux** [johnholbrook/streamdeck-pihole#17]
First of all I really like this plugin it is great :) however it would be nice if you could control more than one pihole from the same button! I have two in my home one is just a secondary backup one that is synced with gravity-sync from the primary one. I don't think this tweak would be hard to add and it would be awesome :)

**Comment by johnholbrook** [comment]
Thanks for your comment!

There is some discussion of this in #4 – you may be able to do what you want with a "multi-action" on the stream deck, although you'd lose the status/stats display on the key.

**Comment by AquaRelliux** [comment]
Yes I already do this but the problem with this approach is as you mentioned the status will be gone. So the button turns kinda dumb you cant know if it turned the pihole on or off by just looking at it since it may already be off and you turn it on by pressing it. I would really like to see this function implemented I think it is a essential function to have because having 2 piholes is almost mandatory to have redundancy on DNS in your network

**Comment by Kwull** [comment]
+1. I'm in same boat :)
